ML TCU Typo (#495)
vatpac-technology/sops: docs/terminal/melbourne.md

When an aircraft requests start clearance, the EN SMC controller shall coordinate with ML TCU to obtain the start clearance.

#### Departures

> [Next](../controller-skills/coordination.md#next) coordination is required from ED ADC to AD TCU for all aircraft **entering AD TCU CTA**.

Reviewer: ```suggestion
[Next](../controller-skills/coordination.md#next) coordination is required from EN ADC to ML TCU for all aircraft **entering ML TCU CTA**.
```

> The Standard Assignable level from ED ADC to AD TCU is:

Reviewer: ```suggestion
The Standard Assignable level from EN ADC to AD TCU is:
```

When an aircraft requests start clearance, the EN SMC controller shall coordinate with ML TCU to obtain the start clearance.

#### Departures

> [Next](../controller-skills/coordination.md#next) coordination is required from ED ADC to ML TCU for all aircraft **entering ML TCU CTA**.

Reviewer: ```suggestion
[Next](../controller-skills/coordination.md#next) coordination is required from ED ADC to AD TCU for all aircraft **entering AD TCU CTA**.
```

> The Standard Assignable level from ED ADC to ML TCU is:

Reviewer: ```suggestion
The Standard Assignable level from ED ADC to AD TCU is:
```
